src/components/Atoms/Controls/Input/index.tsx

```
@@ -56,6 +59,122 @@ const inputClass = tv({
+      break;
+  }
+
+  if (customValidation)
+    rules = { ...rules, ...customValidation } as RegisterOptions;
+
+  return rules;
+};
+
+const createCustomEvent = (
+  name: string,
+  value: any,
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)